.github/workflows/integration.yml

```
@@ -0,0 +1,65 @@
+name: Integration
+
+on:
+  push:
+    branches:
+    - auto
+    - try
+    paths-ignore:
+    # Don't run integration test, when only textfiles are modified
+    - 'rust-toolchain'
+    - 'COPYRIGHT'
+    - 'LICENSE-*'
+    - '**.md'
+
+jobs:
+  toolchain_setup:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: rust-toolchain
+        uses: actions-rs/toolchain@v1.0.3
+        with:
+          toolchain: nightly
+          target: x86_64-unknown-linux-gnu
+          profile: minimal
+      - name: Master Toolchain Setup
+        run: sh setup-toolchain.sh
```

```suggestion
        run: bash setup-toolchain.sh
```
`sh` on Debian based distros points to Dash but this is Bash script.